swagger-spec/search/definition.yaml

```diff
@@ -0,0 +1,197 @@
+type: object
+title: Search
+required:
+  - id
+  - type
+  - attributes
+properties:
+  data:
+    type: array
+    title: Data
+    description: "The search results, each item in this list will be serialized according to it's type."
```

Comment:
Missing semicolon? And additional apostrophe.
`The search results; each item in this list will be serialized according to its type.`